Earthfile

```diff
@@ -62,6 +62,18 @@ build-image:
     ARG tag=latest
     DO core+SAVE_IMAGE --COMPONENT=payments --REPOSITORY=${REPOSITORY} --TAG=$tag
 
+build-worker-image:
+    FROM core+final-image
+    ENTRYPOINT ["/bin/payments"]
+    CMD ["worker"]
+    COPY (+compile/main) /bin/payments
+    FOR c IN $(ls /plugins/*)
+        RUN chmod +x $c
+    END
+    ARG REPOSITORY=ghcr.io
+    ARG tag=latest
+    DO core+SAVE_IMAGE --COMPONENT=payments-worker --REPOSITORY=${REPOSITORY} --TAG=$tag
```

Comment:
why did you need another image ? I think the operator is overriding the args directly, so I don't think we need another image

Reply:
oh I think you're right. I did this before I understood the operator 👍